### DISCUSSION TITLE
layer.open当type=1时content显示内部其他页面

### DISCUSSION THREAD

**Discussion by muquanlin** [layui/layui#1962] [问答]
layer.open({
                        type: 1,
                        offset: 'r',
                        anim: 'slideLeft', // 从右往左
                        area: ['600px', '100%'],
                        shade: 0.1,
                        shadeClose: true,
                        id: 'ID-demo-layer-direction-r',
                        content: '/addData'
                    });

  content: '/addData' :  addData对应项目controller层会返回内部的对应的一个html页面。 type=2这种用法可以，但是当type=1就不行。

**Comment by LSL1618** [comment]
看文档嘛 [https://layui.dev/docs/2/layer/#options.content](https://layui.dev/docs/2/layer/#options.content)
